pairone-core/src/main/java/it/proconsole/utility/pairone/core/model/Pair.java

```diff
@@ -0,0 +1,56 @@
+package it.proconsole.utility.pairone.core.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class Pair {
+  private final String id;
+  private final List<Developer> members;
+
+  @JsonCreator
+  public Pair(String id, List<Developer> members) {
+    this.id = id;
+    this.members = new ArrayList<>(members);
+    this.members.sort((dev1, dev2) -> 0); //TODO
```

Comment:
Don't want to store (A, B) and (B, A) since they are the same Pair, need to understand where to sort and remove duplicates

Reply:
This should be done by https://github.com/ProconsoleCarlo/pairone/issues/5
In case from outside you want to create those cases, you should be free to do it